ros/astra_camera_node.cpp

```diff
@@ -107,16 +117,25 @@ int main(int argc, char **argv){
   double dframerate = 30;
   astra_wrapper::PixelFormat dformat = astra_wrapper::PixelFormat::PIXEL_FORMAT_DEPTH_1_MM;
 
+  bool use_ir = true;
+  bool use_color = true;
+  bool use_depth = true;
+
   // TODO(clalancette): parsing the command-line options here is temporary until
   // we get parameters working in ROS2.
-  if (!parse_command_options(argc, argv, &width, &height, &framerate, &dwidth, &dheight, &dframerate, &dformat)) {
+  if (!parse_command_options(argc, argv, &width, &height, &framerate, &dwidth, &dheight, &dframerate, &dformat, &use_ir, &use_color, &use_depth)) {
     return 1;
   }
 
   rclcpp::init(argc, argv);
   rclcpp::node::Node::SharedPtr n = rclcpp::node::Node::make_shared("astra_camera");
   rclcpp::node::Node::SharedPtr pnh = rclcpp::node::Node::make_shared("astra_camera_");
```

Comment:
Not totally related to this PR but why are we using a private nodehandle here, and mostly why do we spawn 2 nodes o_O?
Parameters should be read on the node we use for spinning IMO (aka "n") https://github.com/ros2/ros_astra_camera/pull/18/files#diff-18aece5c1dfda5045f47c4b87af33d12R141